### PR TITLE
[RPGF-52] Enhance Live Data API

### PR DESCRIPTION
**Pull request by wit03**
# Description

[//]: # "Please include a summary of the changes and the related issue. (You can write below without deleting this comment)"
This pull request introduced changes in run script, added important field to general API response, and add more data to scraping script.

### What
[//]: # "Briefly describe this feature / fix / refactor (You can write below without deleting this comment)"
- Refactor `run script` by removing unused script.
- Add `fs.stats.mtime` to the base api response so client can get mTime for each file (mTime = modified time.)
- feat: projectUid to RetroPGF 5 Scrape.

### Why
[//]: # "Describe the reason why we need this feature / fix / refactor or refer to Jira Ticket (You can write below without deleting this comment)"
- Added modified time to API response can ensure us about the health of our CRON job and also this data can be utilize 

## Type of change

Please check the relevent option with `x`.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if needed)
- [x] My changes generate no new warnings
- [x] I have tested on my `Local Machine`

### CODE REVIEW

File: data/retropgf5-live-data/retropgf5-live-data.json [hunk 1]
```diff
@@ -1,49 +1,538 @@
...
+        "contracts": [],
+        "grantsAndFunding": {
+            "ventureFunding": [],
+            "grants": [],
+            "revenue": []
+        },
+        "pricingModel": "free",
+        "pricingModelDetails": "",
+        "links": [],
+        "projectUid": "0x985a1ed485458d7c383c8b59a18856183197783883014c74b865468769436c52"
```
[the remark this answers is not in the document]
Comment (wit03) [reply data/retropgf5-live-data/retropgf5-live-data.json:225]: These are data from user which we didn't involve in writing process, we can skip this.

File: data/retropgf5-live-data/retropgf5-live-data.json [hunk 2]
```diff
@@ -1,49 +1,538 @@
...
+        "contracts": [],
+        "grantsAndFunding": {
+            "ventureFunding": [],
+            "grants": [],
+            "revenue": []
+        },
+        "pricingModel": "pay_to_use",
+        "pricingModelDetails": "Our price is the lowest fee. Our goal is community before money. It is usually a fee like 0.00001 Eth that everyone can reach.This is true for all the projects I do.",
+        "links": [],
+        "projectUid": "0x6ee546715053790bda72f913925b782461830ccdcc14ad12a2f288641510f448"
```
[the remark this answers is not in the document]
Comment (wit03) [reply data/retropgf5-live-data/retropgf5-live-data.json:37]: These are data from user which we didn't involve in writing process, we can skip this.

File: data/retropgf5-live-data/retropgf5-live-data.json [hunk 3]
```diff
@@ -1,49 +1,538 @@
...
+        "contracts": [],
+        "grantsAndFunding": {
+            "ventureFunding": [],
+            "grants": [],
+            "revenue": []
+        },
+        "pricingModel": "freemium",
+        "pricingModelDetails": "Our price is the lowest fee. Our goal is community before money. It is usually a fee like 0.00001 Eth that everyone can reach.",
+        "links": [],
+        "projectUid": "0x1480383a90f1ab95cfedee6ea130dca700e5236eceb7dcf3f08a0e96b43c90aa"
```
[the remark this answers is not in the document]
Comment (wit03) [reply data/retropgf5-live-data/retropgf5-live-data.json:121]: https://github.com/mali030303/Dragons-On-Op-Stack--

File: src/scripts/retropgf5-live-data/retropgf5-live-data.ts [hunk 4]
```diff
@@ -99,10 +103,21 @@ export async function fetchAndProcessData() {
...
         const concatenatedUrls = [].concat(...urlArrays);
+
         try {
-            const responses = await Promise.all(concatenatedUrls.map(url => axios.get(url)));
-            const dataArray = responses.map(response => response.data);
-            return dataArray;
+            const responses: any[] = [];
+            await Promise.all(concatenatedUrls.map(async (url) => {
+                try {
+                    const response = await axios.get(url.metadataURL);
```
Comment (Billy19191) [inline src/scripts/retropgf5-live-data/retropgf5-live-data.ts:111]: Please add type for response for this `axios.get(url.metadataURL)` (maybe change schema or model in the future)

File: src/scripts/retropgf5-live-data/retropgf5-live-data.ts [hunk 5]
```diff
@@ -99,10 +103,21 @@ export async function fetchAndProcessData() {
             return urls;
         }));
         const concatenatedUrls = [].concat(...urlArrays);
+
         try {
-            const responses = await Promise.all(concatenatedUrls.map(url => axios.get(url)));
-            const dataArray = responses.map(response => response.data);
-            return dataArray;
+            const responses: any[] = [];
```
Comment (Billy19191) [inline src/scripts/retropgf5-live-data/retropgf5-live-data.ts:108]: Add responses type (please avoid any type)

**Review by Billy19191 — CHANGES_REQUESTED** [review]
Other thing look goods, only few comment here

**Comment by wit03** [comment]
@Billy19191 I don't think it's worth wasting time here, since the scraping script will not be used for long (only one month) and also they are not the codebase (unlike RetroPGFHub, and core functionality of Worker in other folders)

Another reason apart from that is we don't have End-2-End typesafe anyway, so by adding type to this wouldn't affect the type on Frontend.

We should better focus on the Worker codebase in other folders apart from `/scripts` 

@TGRZiminiar Do you have any opinion?

**Comment by Billy19191** [comment]
As I understand, this script is temporary, right? At first, I thought that this gonna be permanent in the Worker codebase. So, in this case, you can you with your solution with any type.

**Review by Billy19191 — APPROVED** [review]

**Review by mmxxixx — APPROVED** [review]
Approved

**Pull request merged**